src/UI/class-settings-page.php

```diff
@@ -181,7 +181,7 @@ public function enqueue_settings_assets( ?string $hook_suffix ): void {
 		wp_enqueue_style(
 			'parsely-admin-settings',
 			$built_assets_url . 'admin-settings.css',
-			$admin_settings_asset['dependencies'],
+			array(),
```

Comment:
@acicovic Found the styling issue! We were reusing `$admin_settings_asset['dependencies']` for `wp_enqueue_style()`, which stopped working because that file now has a lot of JS dependencies:

https://github.com/Parsely/wp-parsely/blob/d372a15169ad6c939a57ce312b7338ce97d58084/build/admin-settings.asset.php#L1

These aren't style dependencies, so the dependencies were not available to `wp_enqueue_style()` and the CSS did not get enqueued.

Reply:
Awesome, thank you!